--- a/meson.build
+++ b/meson.build
@@ -6,6 +6,7 @@
   meson_version: '>=0.54.0',
 )
 
+pkg = import('pkgconfig')
 c_args = []
 feature_flags = {
   'reader': 'YYJSON_DISABLE_READER',
@@ -32,7 +33,9 @@
   c_args: [c_args],
   include_directories: yyjson_inc,
   version: meson.project_version(),
+  install: true,
 )
+install_headers('src/yyjson.h')
 
 yyjson_dep = declare_dependency(
   include_directories: yyjson_inc,
@@ -40,3 +43,8 @@
 )
 
 meson.override_dependency('yyjson', yyjson_dep)
+pkg.generate(
+  yyjson_lib,
+  name: 'yyjson',
+  description: 'High performance JSON library',
+)
